website/client/src/app.vue

```diff
@@ -177,6 +179,10 @@ import {
   removeLocalSetting,
 } from '@/libs/userlocalManager';
 
+const bugReportModal = () => import(/* webpackChunkName: "bug-report-modal" */'@/components/bugReportModal');
```

Comment:
I think this should be used in more modals - so that the intial load isn't to huge due to all the other modals (and surely other stuff as well ^^)